src/draw.cpp

```diff
@@ -71,7 +71,7 @@ void GraphicsWindow::Selection::Draw(bool isHovered, Canvas *canvas) {
         auto it = std::unique(refs.begin(), refs.end(),
                               [](Vector a, Vector b) { return a.Equals(b); });
```

Comment:
unrelated but more low hanging fruit: 

```suggestion
                              [](const Vector &a, const Vector &b) { return a.Equals(b); });
```

Reply:
Good catch - thank you.